prettier.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
automatically format code so you dont have to worry about it https://prettier.io/